pkg/genericapiserver/endpoints/openapi/openapi_test.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openapi
+
+import (
+	"testing"
+
+	"github.com/go-openapi/spec"
+	assert_pkg "github.com/stretchr/testify/assert"
```

Comment:
style: Generally we don't use assert unless we have to - it's preferred for tests to not import extra dependencies

Not a huge deal, but I prefer keeping our tests fairly close to Go standard unless there's a big win from the import.